### DISCUSSION TITLE
Modify Existing Network Interfaces - SO 2.3.90

### DISCUSSION THREAD

**Discussion by murph146** [Security-Onion-Solutions/securityonion#6643] [Unsupported Versions] [answered]
How do you modify your existing management/monitoring network interfaces that you setup during initial install of Security Onion 2.3.90. A lot of the documentation I've seen is referring to version 16.04 and haven't had much luck so far making any of this work on 2.3.90? Any help would be appreciated. Thank you.

**Comment by weslambert** [comment, marked as answer]
We don't fully support modifying the management interface (can attempt using `so-ip-update`, but no guarantee it will work correctly):

https://docs.securityonion.net/en/latest/ip.html?highlight=so-ip-update

If you'd like to add an additional monitor interface to the bond used for sniffing, you can use `so-monitor-add`:

https://docs.securityonion.net/en/latest/so-monitor-add.html?highlight=so-monitor-add

**Reply by kepjr2** [reply]
I have a question on this subject, I run several SO management vm's within different domains so domain1, domain2, and domain3 all have an instance of SecurityOnion 2.3.40 running an air gapped build. (This is for a training environment that allows my customers to gain exposure to your product.)
With this build I am able to run 'so-ip-update' successfully which allows me to make clones and redeploy in newly created domains.

Once I uploaded 'securityonion-2.3.91.iso' and ran 'soup' I can no long change the management IP with the command. Does running 'soup' change the type of deployment during the update?

**Reply by dougburks** [reply]
As Wes mentioned above, we don't fully support this and, as mentioned at https://docs.securityonion.net/en/latest/ip.html, `so-ip-update` is an experimental utility.

That being said, I tested updating an airgapped 2.3.40 installation via `securityonion-2.3.91.iso` and everything still works properly including `so-ip-update`.

If you have further questions or problems, please start a new discussion rather than replying to somebody else's discussion (as described in the [discussion guidelines](https://github.com/Security-Onion-Solutions/securityonion/discussions/1720)).
